framework/test/mocha/functional/http/get/peers.js

```diff
@@ -125,8 +125,11 @@ describe('GET /peers', () => {
 			});
 		});
 	});
-
-	describe('pass data from a real peer', () => {
+	/**
+	 * Skipping this GET /api/peers tests as of now because we are using new p2p library and it needs a different apporach to setup the functional test
+	 */
+	/* eslint-disable mocha/no-skipped-tests */
+	describe.skip('pass data from a real peer', () => {
```

Comment:
I don't think it's good approach. It should be quite straightforward to rewrite this file.

Reply:
As discussed with @shuse2 , we decided to resolve these tests in another PR properly, that's why we are skipping it as of now. Also, the problem is not just refactoring from where I can see, the problem is the way we are starting up the test nodes where it is not able to get the peers list from the config file and also it needs a fix from https://github.com/LiskHQ/lisk/pull/3243 related to nodeInfo, so I think this PR should go after rebasing it from it.